lib/publiccloud/instance.pm

```diff
@@ -915,4 +915,13 @@ sub upload_supportconfig_log {
     $self->upload_log('/var/tmp/scc_supportconfig.txz', failok => 1, timeout => 600);
 }
 
+sub wait_for_state {
+    my ($self, $state) = @_;
```

Comment:
Suggestion (feel free to ignore)

This method should have an optional `timeout =>` parameter.